components/alert/alert.css

```diff
@@ -0,0 +1,36 @@
+.alert-area{
+   align-items: center;
+  
+   margin-left: 206px;
+
+}
+.alert{
+
+    border: 1px solid black;
+    padding: 0.5rem;
+    margin: 1rem;
+    text-align: center;
+    width: 15rem;
+    border-radius: 2rem;
+}
+.error{
+    color: #f50057;
+    border-color: #f50057;
+    
+}
+.Success{
```

Comment:
i think  class names should  be lowercase as a best practice , if not then you can ignore it